resonate/resonate.py

```diff
@@ -35,6 +41,7 @@ def __init__(
         pid: str | None = None,
         ttl: int = 10,
         group: str = "default",
+        encoder: Encoder[Any, str | None] | None = None,
```

Comment:
This encoder encodes the param/value data only to make configuring Resonate easier